public/styles/components/_settings-profile-edit.scss

```
@@ -0,0 +1,95 @@
+.settings-profile-edit {
+  &__pfp {
+    margin-bottom: 40px;
+    cursor: pointer;
+    max-width: 110px;
+    position: relative;
+
+    &:hover {
+      svg {
+        transform: scale(1.05);
+      }
+    }
+
+    img {
+      height: 110px;
+      width: 110px;
+    }
+
+    svg {
+      color: $black-grey;
+      fill: $white;
+      height: 35px;
+      position: absolute;
+      right: 0;
+      top: 0;
+      transition: $basic-transition;
+      width: 35px;
+    }
+  }
+
+  &__title {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    position: absolute;
+    top: 20px;
+
+    h3 {
+      color: $white;
+      font-size: 	1.1375rem;
```

There's a lot of space here 😂 
```suggestion
      font-size: 1.1375rem;
```